eZ/Bundle/EzPublishMigrationBundle/Command/LegacyStorage/RegenerateUrlAliasesCommand.php

```diff
@@ -246,7 +245,7 @@ protected function doBackupCustomLocationAliases()
         $customAliasPathCount = 0;
 
         if ($totalCount === 0) {
-            $this->output->writeln('Could not find any Locations, nothing to backup.');
+            $this->output->writeln('Could not find any Locations, nothing to back up.');
```

Comment:
imho 'to back up' is a different verb than 'to backup'

Reply:
"Back up" is a verb, "backup" a noun. So Merriam Webster, MMOS and others.

Reply:
Indeed, I was wrong on this one. I did find on google some evidence supporting my version yesterday, but it is dwarfed by the consensus about 'backup' not being acceptable as verb.
It is just my programmer's habit of thinking about 'back up' as non-IT verb for getting support or moving backwards, and abusing (verbying ?) the 'backup' word in order to make it clear that it is to be understood in its IT-related meaning.
OT: funny little site notaverb.com has a long list of phrasal verbs often abused in IT contexts :-)